general_motion_retargeting/motion_retarget.py

```diff
@@ -64,24 +66,21 @@ def __init__(
             ratio = actual_human_height / ik_config["human_height_assumption"]
         else:
             ratio = 1.0
-            
-        # adjust the human scale table
-        for key in ik_config["human_scale_table"].keys():
-            ik_config["human_scale_table"][key] = ik_config["human_scale_table"][key] * ratio
-    
 
+        self.configuration = mink.Configuration(self.model)
+    
         # used for retargeting
         self.ik_match_table1 = ik_config["ik_match_table1"]
         self.ik_match_table2 = ik_config["ik_match_table2"]
         self.human_root_name = ik_config["human_root_name"]
         self.robot_root_name = ik_config["robot_root_name"]
         self.use_ik_match_table1 = ik_config["use_ik_match_table1"]
         self.use_ik_match_table2 = ik_config["use_ik_match_table2"]
-        self.human_scale_table = ik_config["human_scale_table"]
+        self.human_scale_table = self.build_human_scale_table(ik_config, ratio)
```

Comment:
Remnant. I made a helper function for this for some visualization code that I deleted.